modules/nw-networkpolicy-about.adoc

```diff
@@ -6,8 +6,8 @@
 
 = About network policy
 
-In {product-title} {product-version} `NetworkPolicy` is the default mode for the
-`OpenShiftSDN` plug-in for the Kubernetes Container Network Interface (CNI).
+In {product-title} {product-version}, `NetworkPolicy` is the default mode for
```

Comment:
default mode - what is a mode? What dos network policy do? Why do we care about this?